app/views/layouts/user_mailer.html.erb

```diff
@@ -71,7 +71,7 @@
                                         </tr>
                                       </tbody>
                                     </table>
-                                    <% unless @hide_title %>
+                                    <% unless @hide_title or @pending_profile_requires_verification %>
```

Comment:
Instead of adding an extra condition here and making the base template aware of a specific use-case, could you assign `@hide_title` in the `reset_password_instructions` method?

```ruby
@hide_title = @pending_profile_requires_verification
```